zookeeper-server/src/main/java/org/apache/zookeeper/server/watch/WatcherCleaner.java

```diff
@@ -177,6 +180,7 @@ public void shutdown() {
         stopped = true;
         deadWatchers.clear();
         cleaners.stop();
+        super.interrupt();
```

Comment:
Maybe just change it to this.interrupt(), since we're not overwriting it.

Reply:
Sure.Used super for better readability.

Reply:
Done